File: April-18/19-04-18/python/01-Jumping-Chumpa.py
for _ in range(int(input())):
    n, q = map(int, input().split())
    ar = [int(x) for x in input().split()]
    ar.sort()
    
    ans = 0
    
    for i in range(0, n-1):
        ans += (q*abs(ar[i]-ar[i+1]))
    print(ans)

File: April-18/19-04-18/java/01-Jumping-Chumpa.java

import java.util.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();
        while(t-- > 0)
        {
            long n, q;
            n = input.nextLong();
            q = input.nextLong();
            long ar[] = new long[(int)n];
            for(int i = 0; i < n; i++)
                ar[i] = input.nextLong();
            
            Arrays.sort(ar);
            long ans = 0;
            for(int i = 0; i < n-1; i++)
            {
                ans += (q*Math.abs(ar[i]-ar[i+1]));
            }
            System.out.println(ans);
        }
        
    }
}
